src/ccnl-pkt/src/ccnl-pkt-ndntlv.c

```diff
@@ -115,8 +115,6 @@ ccnl_ndntlv_bytes2pkt(uint64_t pkttype, uint8_t *start,
 #endif
 
 
-    write(1, *data, *datalen);
```

Comment:
I agree that this seems unnecessary. Instead of writing it to ``stdout``, a log statement should have been used and probably set to ``TRACE`` level.